lib/Db/AssignedLabels.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @copyright Copyright (c) 2020 Jakob Röhrl <jakob.roehrl@web.de>
+ *
+ * @author Jakob Röhrl <jakob.roehrl@web.de>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Deck\Db;
+
+use JsonSerializable;
+
+class AssignedLabels extends RelationalEntity implements JsonSerializable {
+	public $id;
+	protected $label;
+	protected $cardId;
+
+	public function __construct() {
+		$this->addType('id', 'integer');
+		$this->addType('cardId', 'integer');
+		$this->addResolvable('label');
```

Comment:
The database returns a field that is called `label_id` so we need a property for that one as well.